companies/she-explores.md

```diff
@@ -0,0 +1,26 @@
+---
+name: She Explores
+blm_statements:
+  - url: https://www.facebook.com/sheexplores/photos/a.699532496805536/3077810698977692/?type=3
+    date_posted: 2020-05-31
+    type: Facebook
+    raw_text: >
+      Racism had carved a well-worn, dangerous trail in this country. Its roots are long and deep, but we can see it right under our feet, within our systems, and within ourselves.⁣
+
+      On She Explores we often talk about the growth we experience when we’re outdoors, but we don’t talk enough about the fact that not everyone feels safe outside due to pervasive and systemic racism. It shouldn’t take more black lives for us to say that and take action. We can do better, you can do, too. This conversation is also part of growth.⁣
+
+      The outdoors doesn’t exist in a vacuum. It’s out our front doors just as much as it’s in the backcountry. It’s every stop along the way. Black people, (as well as indigenous folks and other people of color) don’t just fear for their own lives outdoors, they fear for their family and friends’ safety, too. Trauma is cumulative and painful. Its effects are long lasting and generational. Standing by is unacceptable.
+
+      We’re big proponents of listening, but we challenge those of you who benefit from privilege to take it one step further: be an active listener. Engage. Learn from the advice and experiences of Black experts, but don’t expect it. Pay for it when it’s feasible. Vote. Speak out. Hold conversations with loved ones. Support the work of organizations and businesses like @melaninbasecamp, @blackgirlstrekkin, @browngirlsclimb, @outdoorafro, @blackpeoplewhohike, @girltrek, @outdoorceopledge & @outdoorjournaltour.
+
+      We hope you’ll join us in honoring black lives by acknowledging that racism exists and actively striving to exude its opposite.⁣
+
+      #blacklivesmatter #georgefloyd #breonnataylor #armaudarbery #christiancooper
+exec_team_url: https://she-explores.com/about/
+diversity_inclusion_url: https://she-explores.com/about/
+additional_resources:
+  - url: https://she-explores.com/podcast/bonus-episode-call-to-action-black-lives-matter/
```

Comment:
ooo these look good!! Nice finds 🧗🏻‍♀️

Reply:
Thanks! I don't know when that episode was posted, but it was too relevant to ignore, so I decided to add it as an additional resource.